vimrc

```diff
@@ -93,6 +93,9 @@ set autoindent
 set noswapfile
 set mouse=a
 
+set cursorcolumn
```

Comment:
rendering this is linear with your screen size - have you considered not creating such an egregious performance regression?

alternatively, build vim in a docker container using nodejs so users can `brew install apt-get` and then `yum emerge vim --fast` and get your fix